src/css/components/actions.scss

```diff
@@ -25,44 +24,104 @@
   }
 
   &.action-cautious {
-    background-color: $color-darkgray;
-
-    &:hover { color: $color-primary; }
+    background-color: $color-lightgray;
 
     &.action-link {
       background-color: transparent;
-      color: $color-gray;
+      color: $color-lightgray;
     }
 
     &.usa-button-outline {
       background-color: transparent;
-      color: $color-gray;
-      box-shadow: inset 0 0 0 1px $color-gray;
+      box-shadow: inset 0 0 0 1px $color-lightgray;
+      color: $color-lightgray;
       font-size: $sans-s2;
-
-      &:hover {
-        box-shadow: inset 0 0 0 1px $color-primary;
-      }
     }
   }
 
   &.action-warning {
     background-color: $color-warning;
 
+    &:hover {
+      background-color: $color-textblack;
+    }
+
     &.action-link {
       background-color: transparent;
       color: $color-warning;
+      &:hover {
+        color: $color-textblack;
+      }
     }
 
     &.usa-button-outline {
-      background-color: transparent;
-      color: $color-warning;
+      background-color: $color-white;
       box-shadow: inset 0 0 0 1px $color-warning;
+      color: $color-warning;
+
+      &:hover {
+        box-shadow: inset 0 0 0 1px $color-textblack;
+        color: $color-textblack;
+      }
     }
   }
 
   &.action-finish {
     background-color: $color-finish;
+
+    &:hover {
+      background-color: $color-textblack;
+      color: $color-white;
+    }
+
+    &.action-link {
+      background-color: transparent;
+      color: $color-finish;
+
+      &:hover {
+        color: $color-textblack;
+      }
+    }
+
+    &.usa-button-outline {
+      background-color: $color-white;
+      box-shadow: inset 0 0 0 1px $color-finish;
+      color: $color-finish;
+
+      &:hover {
+        box-shadow: inset 0 0 0 1px $color-textblack;
+        color: $color-textblack;
+      }
+    }
+  }
+
+  &.action-base {
```

Comment:
The one place I can think of where there's a base action state is in the breadcrumb nav, where the trigger buttons start in a base state.